### DISCUSSION TITLE
ONLY works with GPT-4 access??

### DISCUSSION THREAD

**Discussion by GSCrawley** [AntonOsika/gpt-engineer#175] [Q&A]
Where are you declaring the Model? I don't see it anywhere, and I am still waiting for access to the GPT-4 API. 
From what I understand of late the 3.5 turbo API is pretty much just as good anyways no? At any rate, I'd really like to try this out. How can I change the model to 3.5-turbo?

**Comment by patillacode** [comment]
When you run GPT Engineer with a Open AI API Key you should get a message like this if you key doesn't support gpt-4

```
Model gpt-4 not available for provided API key. Reverting to gpt-3.5-turbo. Sign up for the GPT-4 wait list here: https://openai.com/waitlist/gpt-4-api
```

So, to answer your question, there is a fallback value for the model

**Comment by GSCrawley** [comment]
yes, I gathered that, but how much of a difference does it actually make
when you have a key that supports gpt-4? can the model actually perform
effectively on 3.5-turbo?
